src/docs/development/tools/devtools/memory.md

```diff
@@ -149,86 +185,139 @@ in the legend.
 
 For more information, see [Dart VM internals][vm].
 
-## Event timeline
+## Memory Android Chart (ADB)
+
+DevTools, for an Android app, collects Android's ADB tool parsing the meminfo section of ADB's App Summary section.  This section is the most interesting from a high-level
+```
+> adb shell dumpsys meminfo io.flutter.demo.gallery -d
+
+ App Summary
+                       Pss(KB)
+                       -------
+           Java Heap:     5192
+         Native Heap:    11992
+                Code:     2132
+               Stack:       60
+            Graphics:    53700
+       Private Other:    42800
+              System:    84493
+ 
+               TOTAL:   200369       TOTAL SWAP PSS:    82168
+```
+
+This chart is another timeseries graph but of Android memory (polled ADB values).  to help visualize the state of the
+heap at various points in time.
+
+The chart's x-axis is a timeline of events (timeseries).
+In other words, it shows the polled state of the memory
+every 500 ms. This helps give a live appearance on the
+state of the memory as the application is running.
+The quantities plotted on the y-axis are (from top to bottom):
+
+## Event Timeline
 
-![Screenshot of DevTools events]({% asset tools/devtools/memory_snapshot_reset.png @path %}){:width="100%"}
+![Screenshot of DevTools events]({% asset tools/devtools/memory_eventtimeline.png @path %}){:width="100%"}
 
-This chart displays DevTools events (such as Snapshot and Reset button
+This chart displays DevTools events (such as manual GC, VM GC, Snapshot, Monitor Allocations and Reset Accumulators button
 clicks) in relation to the memory chart timeline. Hovering over the
 markers in the Event timeline displays the time when the event occurred.
 This helps identify when a memory leak might have occurred in the
 timeline (x-axis).
 
-![Screenshot of the event timeline]({% asset tools/devtools/memory_eventtimeline.png @path %}){:width="100%"}
+![Screenshot of the event timeline legend]({% asset tools/devtools/memory_eventtimeline_legend.png @path %}){:width="100%"}
 
-Clicking the **Snapshot** button shows the current state of the heap with
-regard to all active classes and their instances. When the **Reset** button
-is pressed, the accumulator for all classes resets to zero.
-The reset is temporally tied, using a faint blue horizontal bar,
-to the previous snapshot. Clicking the **Reset** button again resets the
-accumulators since the last reset and temporally ties the latest
-reset to the previous reset.
+The above list of glyphs plotted on the Event Timeline corresponds to users actions or a VM has garbaged collected.
 
-## Snapshot classes
+<dl markdown="1">
```

Comment:
a lot of this information feels duplicated with the above the descriptions. For the ones that don't add any extra information (e.g. snapshot, etc.), maybe a simple "User initiated snapshot." description would suffice